Patterns/Turkish/Turkish-Numbers.yaml

```diff
@@ -0,0 +1,455 @@
+FractionPrepositionRegex2: !nestedRegex
+  def: (?<=\b)(?<denominator>((?!yüzde){AllDenominatorSuffixRegex}|(?<![\.,]){NumbersDenominatorSuffixRegex}))\s(?<numerator>({AllIntRegex}|{NegativeAllIntRegex})|(\d+)(?![\.,]))(?=\b)
+  references: [ AllDenominatorSuffixRegex, NumbersDenominatorSuffixRegex, AllIntRegex, NegativeAllIntRegex ]
+#Double Regex
+AllPointRegex: !nestedRegex
+  def: ((\s{ZeroToNineIntegerRegex})+|(\s{AllIntRegex}))
+  references: [ ZeroToNineIntegerRegex, AllIntRegex ]
+FloatRegex1: !nestedRegex
+  def: (({AllIntRegex}|{NegativeAllIntRegex})(\s(virgül|nokta)){AllPointRegex})
+  references: [ AllIntRegex, AllPointRegex, NegativeAllIntRegex ]
+FloatRegex2: !nestedRegex
+  def: '{AllIntRegex}(\s+(tam)\s+)((onda)\s+{OneToNineIntegerRegex}|(yüzde)\s+({OneToNineIntegerRegex}|{TenToHundredRegex})|(binde)\s+({OneToNineIntegerRegex}|{TenToHundredRegex}|{HundredToThousandRegex}))'
+  references: [ AllIntRegex, OneToNineIntegerRegex, TenToHundredRegex, HundredToThousandRegex]
+AllFloatRegex: !nestedRegex
+  def: ({FloatRegex1}|{FloatRegex2})
+  references: [FloatRegex1, FloatRegex2]
+DoubleWithMultiplierRegex: !nestedRegex
+  def: (?<=\b)(?<!\d+,)\d+,\d+\s*({BaseNumbers.NumberMultiplierRegex}|{BaseNumbers.MultiplierLookupRegex})(?=\b)
+  references: [ BaseNumbers.NumberMultiplierRegex, BaseNumbers.MultiplierLookupRegex ]
+DoubleExponentialNotationRegex: !simpleRegex
+  def: (((?<!\d+\s*)-\s*)|((?<=\b)(?<!\d+,)))(\d+(,\d+)?)e([+-]*[1-9]\d*)(?=\b)
+DoubleCaretExponentialNotationRegex: !simpleRegex
+  def: (((?<!\d+\s*)-\s*)|((?<=\b)(?<!\d+,)))(\d+(,\d+)?)\^([+-]*[1-9]\d*)(?=\b)
+DoubleDecimalPointRegex: !paramsRegex
+  def: (((?<!\d+\s*)-\s*)|((?<=\b)(?<!\d+\,)))\d+\,\d+(?!(\,\d+))(?={placeholder})
+  params: [ placeholder ]
+DoubleWithoutIntegralRegex: !paramsRegex
+  def: (?<=\s|^)(?<!(\d+)),\d+(?!(,\d+))(?={placeholder})
+  params: [ placeholder ]
+DoubleWithRoundNumber: !nestedRegex
+  def: (((?<!\d+\s*)-\s*)|((?<=\b)(?<!\d+,)))\d+,\d+\s+{RoundNumberIntegerRegex}(?=\b)
+  references: [ RoundNumberIntegerRegex ]
+DoubleAllFloatRegex: !nestedRegex
+  def: ((?<=\b){AllFloatRegex}(?=\b))
+  references: [ AllFloatRegex ]
+#Percentage Regex
+NumberWithPrefixPercentage: !nestedRegex
+  def: (%|(eksi\s)?yüzde\s)({BaseNumbers.NumberReplaceToken}|{AllIntRegex})
+  references: [ BaseNumbers.NumberReplaceToken, AllIntRegex ]
+NumberWithSuffixPercentage: !nestedRegex
+  def: (((({TensNumberIntegerRegex}\s)?(birin|ikinin|üçün|dördün|beşin|altının|yedinin|sekizin|dokuzun)|onun|yirminin|otuzun|kırkın|ellinin|altmışın|yetmişin|seksenin|doksanın)\s(yüzdesi))|(\d*(1'in|2'nin|3'ün|4'ün|5'in|6'nın|7'nin|8'in|9'un|10'un|20'nin|30'un|40'ın|50'nin|60'ın|70'in|80'in|90'ın)\s(yüzdesi)))
+  references: [ TensNumberIntegerRegex ]
+FractionNumberWithSuffixPercentage: !simpleRegex
+  def: (\d+)\s+\/(\d+)?((1|5|8|70|80|\.?000)'i|(2|7|20|50)'si|(3|4|100)'ü|6'sı|(9|10|30|\.?000\.?000|\.?000\.?000\.?000\.?000)'u|(40|60|90|\.?000\.?000\.?000)'ı)
```

Comment:
Why are these needed? "\.?000\.?000|\.?000\.?000\.?000\.?000"

Reply:
In Turkish, numbers get suffixes and it depends on the numbers so if it ends with thousands (.000), it get a certain suffix etc cases like "1 bölü 1000'i" meaning "one over 1000". 
This is solved elsewhere now.